package.json

```diff
@@ -7,7 +7,7 @@
     "start": "gulp",
     "build": "yarn install && gulp generate-assets && gulp sync",
     "unit-test": "standard && mocha --recursive test/unit/",
-    "integration-test": "gulp assets && mocha --recursive test/integration --timeout=10000",
+    "integration-test": "gulp assets && mocha --recursive test/integration --timeout=30000",
```

Comment:
The data inserts and removes were timing out on me. Not sure if it's maybe because I'm on a worse network when WFH or something.